adhoc().catch((e) => {
  console.error(e);
  process.exit(1);
});

async function adhoc() {
  console.log('beginning adhoc...');
}
